api-examples/pretrain-paired.py

```diff
@@ -55,6 +55,7 @@ def train():
     parser.add_argument("--stacking_layers", type=int, nargs='+', default=[1024, 1024, 1024],
                         help="Hidden sizes of the dense stack (ff2 from the convert paper)")
     parser.add_argument("--ff_pdrop", type=float, default=0.1, help="Dropout in the dense stack")
+    parser.add_argument("--ra_masking", type=str2bool, default=False, help="whether prevent attention beyond rpr_k")
```

Comment:
this term doesnt make a lot of sense to me, what about `windowed_ra` or `local_ra` or something that is more descriptive of the purpose